__tests__/app.test.js

```diff
@@ -159,6 +159,7 @@ describe("App", () => {
           expect(article.title).toBe("Eight pug gifs that remind me of mitch");
           expect(article.article_id).toBe(3);
           expect(article.author).toBe("icellusedkars");
```

Comment:
We shouldn't amend existing tests. The existing tests should still pass if they are robust enough (which they should be) Instead this should be a whole new test whose only job will be to check if there is now a comment_count key/value pair

Reply:
done